Azure/sonic-buildimage
# [device/celestica]: Fix sfp index on dx010 platform api

**- What I did**
- Fixed port indexing issue in DX010 sfp api

**- How I did it**
- Used overriding method to fix the port index issue.

**- How to verify it**
- Check sfpshow presence and verify port status is getting updated or not
- Check sfpshow eeprom and verify port eeprom is getting updated or not
- Test log : [dx0101_sfpshow_logs.txt](https://github.com/Azure/sonic-buildimage/files/4414590/dx0101_sfpshow_logs.txt)
